### DISCUSSION TITLE
Run setup.py to install METplus and other packages?

### DISCUSSION THREAD

**Discussion by lesnow93** [dtcenter/METplus#1802] [Installation] [answered]
I’m confused that the user’s guide doesn’t mention anything about installing the package using setup.py, nor does the README.md say anything.  Do I need to run “python setup.py” to officially install METplus (and the other companion packages such as METplotpy, METviewer, etc.)?  

Thank you,
Jonathan Case

**Comment by JohnHalleyGotway** [comment, marked as answer]
@lesnow93, METplus is a set of python wrappers for which no 'installation' is required, or than checking out the repository or downloading a release tarfile. That differs greatly from the compiled MET C++ code, which obviously does require running configure/make install.

It sounds like you may be just getting started with METplus. Let me provide some links to info you may find useful:
- [Installation Chapter](https://metplus.readthedocs.io/en/develop/Users_Guide/installation.html) of the METplus User's Guide
- [METplus Setup](https://dtcenter.org/metplus-practical-session-guide-version-4-0/session-1-metplus-setupgrid-grid/metplus-setup) section of the online tutorial
- [Session 1](https://dtcenter.org/events/2021/metplus-training-series/agenda-recordings) of the recent METplus training series talks about getting started

Hope those references help. If you have more questions, I'll probably refer them to @jprestop, who handles most of the METplus-related installs. She's out of town the rest of this week, but will be back next week.

**Reply by lesnow93** [reply]
Thanks John H-G!  It appears I'm good on the install and config of both MET and METplus, so I'll be going through the use cases and tutorial content to get set up for running my own verification experiment through METplus.

**Reply by JohnHalleyGotway** [reply]
@lesnow93, great, I'll go ahead and mark this discussion as being answered. And I'll lock it to prevent future posts. That's how we encourage users to create new discussions for new questions. So please feel free to create a new one if you run into issues.

Thanks!
